### PR TITLE
[Model] DistilBERT

### PR DESCRIPTION
**Pull request by eric-haibin-lin**
## Description ##
The distilBERT model from https://arxiv.org/abs/1910.01108 

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by mli** [comment]
Job PR-922/1 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-922/1/index.html

**Comment by mli** [comment]
Job PR-922/5 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-922/5/index.html

**Comment by mli** [comment]
Job PR-922/6 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-922/6/index.html

**Comment by mli** [comment]
Job PR-922/7 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-922/7/index.html

**Comment by mli** [comment]
Job PR-922/8 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-922/8/index.html

**Comment by mli** [comment]
Job PR-922/9 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-922/9/index.html

File: scripts/bert/conversion_tools/convert_pytorch_transformers.py [hunk 1]
```diff
@@ -0,0 +1,211 @@
+# coding: utf-8
```
Comment (szha) [inline scripts/bert/conversion_tools/convert_pytorch_transformers.py]: How do you intend to maintain and verify this script?

Comment (eric-haibin-lin) [reply scripts/bert/conversion_tools/convert_pytorch_transformers.py]: They're currently not maintained, and not documented. Ideally we should test them in CI (adding fairseq, tf, pytorch transformer). But I don't have the bandwidth to do that

**Comment by mli** [comment]
Job PR-922/10 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-922/10/index.html

**Comment by mli** [comment]
Job PR-922/11 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-922/11/index.html

File: scripts/conversion_tools/convert_pytorch_transformers.py [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+# 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint:disable=redefined-outer-name,logging-format-interpolation
+""" Script for converting the distilbert model from pytorch-transformer to Gluon.
+
+Usage: 
+
+pip3 install pytorch-transformers
```
Comment (leezu) [inline scripts/conversion_tools/convert_pytorch_transformers.py]: That package seems not maintained anymore. Why not convert from transformers package?

Comment (eric-haibin-lin) [reply scripts/conversion_tools/convert_pytorch_transformers.py]: At the time of writing this script, there was only the pytorch-transformer package. I think it's still useful to some users, who started to use bert with `pytorch-transformers`.

**Comment by mli** [comment]
Job PR-922/13 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-922/13/index.html

**Comment by mli** [comment]
Job PR-922/12 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-922/12/index.html

**Comment by mli** [comment]
Job PR-922/14 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-922/14/index.html

File: scripts/index.rst [hunk 3]
```diff
@@ -69,6 +69,18 @@ Model Zoo
 
       Predicting the intent of the query and extracting semantic concepts in the query.
 
+   .. card::
+      :title: Sequence Labeling
+      :link: sequence_labeling/index.html
```
Comment (leezu) [inline scripts/index.rst]: Not included in this PR?

**Comment by mli** [comment]
Job PR-922/15 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-922/15/index.html

**Comment by cnlewis3** [comment]
@mli Super happy DistilBERT was added as a model option!
Is exporting symbols file from the distilbert model an option? It looks like the only choices for that are the base and large bert models. Wasn't sure if I should open an issue to ask.

**Comment by leezu** [comment]
It's a `HybridBlock`, so you can call `.export()` to get the symbol

**Pull request merged**